src/test/java/hw4/base/BaseTests.java

```diff
@@ -0,0 +1,256 @@
+package hw4.base;
+
+import hw4.beans.Color;
+import hw4.pages.DifferentElementsPage;
+import hw4.pages.HomePage;
+import hw4.pages.SecondFrame;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.asserts.SoftAssert;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.Properties;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class BaseTests {
+    protected WebDriver driver;
+    protected Properties property = new Properties();
+    SoftAssert softAssert;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "D:/chromedriver_win32/chromedriver.exe");
+        driver = new ChromeDriver(getChromeOptions());
+        softAssert = new SoftAssert();
+        try {
+            property.load(
+                    new InputStreamReader(
+                            new FileInputStream("src/test/resources/hw3config.properties"),
+                            Charset.forName("UTF-8")));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
hint: `java.util.resourcebundle` can be used

Reply:
rewrite some methods using java.util.resourcebundle